__docs__/wonder-blocks-icon-button/icon-button.stories.tsx

```diff
@@ -47,61 +94,41 @@ export const Default: StoryComponentType = {
     },
 };
 
-export const Basic: StoryComponentType = () => {
-    return (
-        <IconButton icon={icons.search} onClick={() => console.log("Click!")} />
-    );
-};
-
-Basic.parameters = {
-    docs: {
-        description: {
-            story: `Minimal icon button. The only props specified in
-            this example are \`icon\` and \`onClick\`.`,
-        },
-    },
-};
-
 /**
- * Icon buttons can be one of three sizes: `xsmall` (16px icon with a 24px touch target),
- * `small` (24px icon with a 32px touch target), and `medium` (24px icon with a 40px touch target).
- * The default size is `medium`.
+ * IconButtons can be used with any icon from the `@phosphor-icons/core`
+ * package. The `icon` prop takes an SVG asset from the package.
+ *
+ * In this example you can see the different sizes of the icon button:
+ * - `xsmall` (16px icon with a 24px touch target).
+ * - `small` (24px icon with a 32px touch target).
+ * - `medium` (24px icon with a 40px touch target).
  */
-export const Sizes: StoryComponentType = (() => {
-    return (
-        <View style={{flexDirection: "column"}}>
+export const Sizes: StoryComponentType = {
```

Comment:
**note:** Refactored this story to present better docs formatting and take advantage of CSF v3 (Spreading stories).